test/run_pass/test_uart.js

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var Uart = require('uart');
+
+function sleep(n) {
+    var d = new Date();
+    while(new Date() - d < n) {}
+}
+
+
+
+var uart = new Uart("/dev/ttyUSB0", {}, function(err){
+  if (err) {
+    throw err;
+  }
+
+  uart.write("Hello? from Ubuntu\n", function(err) {
+  });
+
+  uart.on('data', function(data) {
+    console.log(data.toString());
+  });
+
+});
```

Comment:
I can't see call ``open`` method directly. Is it needed to ``open`` method as ``public``?